### PR TITLE
Completed homework 10

### PR DESCRIPTION
**Pull request by LastStap**

### CODE REVIEW

File: src/main/java/Dragons/Dragon.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        this.age = age;
+        this.weight = weight;
+    }
+
+    public void setName(String name){
+        this.name = name;
+    }
+
+        public void setAge(int age){
+            this.age = age;
```
Comment (nikmikhailov13) [inline src/main/java/Dragons/Dragon.java:30]: nit: formatting

File: src/main/java/Dragons/Dragon.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+        public void setWeight(int weight){
+            this.weight = weight;
```
Comment (nikmikhailov13) [inline src/main/java/Dragons/Dragon.java:34]: nit: same here

**Review by nikmikhailov13 — APPROVED** [review]
LGTM

File: src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -1,8 +1,92 @@
...
+
+        dataDragons.add(String.valueOf(dragon));
+
+        System.out.println("Dragon added: " + dragon);
+
+        whatDragonDo(dragon);
+    }
+
+
+    static void whatDragonDo(Dragon dragon) {
```
Comment (nikmikhailov13) [inline src/main/java/org/example/Main.java:35]: these functions should be part of the Dragon class

Comment (nikmikhailov13) [reply src/main/java/org/example/Main.java:35]: but very interesting :)

File: src/main/java/Dragons/Dragon.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    public int getAge() {
+        return this.age;
+    }
+
+    public int getWeight() {
+        return this.weight;
+    }
+
+    public String getType(){
```
Comment (mykola-digtiar) [inline src/main/java/Dragons/Dragon.java:53]:
```suggestion
    public String getType() {
```

File: src/main/java/Dragons/dragonTypes.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package Dragons;
+
+public enum dragonTypes {
```
Comment (mykola-digtiar) [inline src/main/java/Dragons/dragonTypes.java:3]: Класи ми називаємо `CamelCase` та не через plural i.e. just `DragonType`.
```suggestion
public enum DragonType {
```

**Pull request merged**